### PR TITLE
🐛 Clearing locations on change bug

### LINKED ISSUE #169: Locations not cleared on change to clusters
**Describe the bug**
Locations added to the map from the desktop list view are not cleared from the map when zooming out into cluster view.

**To Reproduce**

(in desktop mode)
1. Zoom in on a city with lots of locations (Champaign, IL) until location markers are displayed.
2. Zoom back out until clusters are displayed.

**Expected behavior**
All locations are cleared from the map when clusters are shown.

**Screenshots**
image

### PR DESCRIPTION
**Pull request by riyajain59**
## Status:

:rocket: Ready
## Description
Initially, locations added to the map from the desktop list view are not cleared from the map when zooming out into cluster view. Just fixed this issue such that the locations are cleared when zooming out to clusters on the map. They do not appear at the same time anymore.

Fixes #169

### CODE REVIEW

**Comment by Qrtn** [comment]
@riyajain59 could you tag the issue and remove some of the blank sections in the PR description? Thanks!!

File: src/redux/viewChange.js [hunk 1]
```diff
@@ -41,7 +44,6 @@ export const fetchLocations = () => (dispatch, getState) => {
       dispatch(clearListLocations())
     } else {
       dispatch(fetchMapLocations())
-
```
Comment (code-arman) [inline src/redux/viewChange.js:44]: Should we keep the space here?

**Review by code-arman — APPROVED** [review]
Looks great Riya 💃. Can you add the PR status to the description as well? Also, nice emoji 💪

**Review by daniel-moon32 — APPROVED** [review]
Looks good!

**Pull request merged**